.github/workflows/rust.yml

```diff
@@ -2,14 +2,19 @@ name: Rustlings Tests
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
I intentionally used `actions/checkout@v4` here because `actions/checkout@v3` uses a deprecated `Node.js` version.

Reply:
I'm happy to take a look at that in a separate pull request.